python/dlr/dlr_model.py

```diff
@@ -103,26 +97,15 @@ class DLRModelImpl(IDLRModel):
     dev_id : int
         Device ID
     """
+    
+    # TVM crashes with `Check failed: override: Global PackedFunc runtime.module.loadfile_so is already registered`
+    # when we try to link muliple versions of the dlr withing the same process. In order to prevent this we make 
+    # _LIB a class attribute which is instantiated only once per class.
```

Comment:
Lets replace this comment with the following:
```
# `libdlr.so` will be loaded when the first `DLRModelImpl` instance is created.
# Once `libdlr.so` is loaded it will be used to run the first and all consequent models.
```